### ISSUE TITLE
UIExplorer is broken: undefined is not an object (evaluating 'scenes.map')

### ISSUE THREAD

**Issue by PublicParadise** [facebook/react-native#6251]
#### Problem

UIExplorer seems to be broken. I am getting this error on launch:

<img width="468" alt="screen shot 2016-03-02 at 2 16 43 pm" src="https://cloud.githubusercontent.com/assets/2541332/13477632/383c2ecc-e082-11e5-8056-2d91dc7460a4.png">

#### Environment

- OSX 10.10.5
- XCode 7.4
- iPhone 6 simulator
- React Native version = 8717b2d19cf7fbe8898a039507f8f41d1c7baa06

#### Steps

```bash
# Big Bang: start from a clean build
watchman watch-del-all
rm -fr $TMPDIR/react-*
rm -rf react-native
git clone https://github.com/facebook/react-native.git
cd react-native 
npm install
open Examples/UIExplorer/UIExplorer.xcodeproj
```

#### Console

```
2016-03-02 14:14:06.565 [info][tid:com.facebook.React.JavaScript] Running application "UIExplorerApp" with appParams: {"rootTag":1,"initialProps":{}}. __DEV__ === true, development-level warning are ON, performance optimizations are OFF
2016-03-02 14:14:06.581 [warn][tid:com.facebook.React.JavaScript] Warning: Failed propType: expectedClass is not a function. (evaluating 'props[propName] instanceof expectedClass') Check the render method of `NavigationComponent`.
2016-03-02 14:14:06.634 [warn][tid:com.facebook.React.JavaScript] Warning: Failed propType: Required prop `scenes` was not specified in `NavigationHeader`. Check the render method of `NavigationComponent`.
2016-03-02 14:14:06.635 [warn][tid:com.facebook.React.JavaScript] Warning: Failed propType: Required prop `index` was not specified in `NavigationHeader`. Check the render method of `NavigationComponent`.
2016-03-02 14:14:06.636 [error][tid:com.facebook.React.JavaScript] undefined is not an object (evaluating 'scenes.map')
2016-03-02 14:14:06.638 [fatal][tid:com.facebook.React.RCTExceptionsManagerQueue] Unhandled JS Exception: undefined is not an object (evaluating 'scenes.map')
```

**Comment by yejodido** [comment]
Same issue here; found this looking for a solution.

**Comment by brandonfancher** [comment]
It works a few commits back. I checked out an earlier commit from today (eb531943aab37ec59c9e160f8b602313b4aa954f), ran `rm -rf node_modules && npm install`, and it seems to be working. Latest commit (2708df4) is broken for me. I haven't tried commits in between to see when, exactly, it broke.

**Comment by dinghua** [comment]
+1

**Comment by mysoniw** [comment]
I have the same issue ... 2708df4

**Comment by megazoid84** [comment]
I have the same problem

**Comment by yimiaodaren** [comment]
+1

**Comment by PublicParadise** [comment]
Seems to be fixed now: 87245b2d40a865290fbeb4d8f5474fb8b5c1b891
Closing...
